src/Github/Api/GraphQL/Query/GetMilestoneChangelog/Response/Label.php

```diff
@@ -37,17 +37,16 @@ private function __construct(
      *
      * @psalm-pure
      *
-     * @psalm-suppress ImpureMethodCall the {@see UriInterface} API is pure by design
+     * @psalm-suppress ImpureMethodCall     {@see https://github.com/azjezz/psl/issues/130}
+     * @psalm-suppress ImpureFunctionCall   {@see https://github.com/azjezz/psl/issues/130}
      */
     public static function fromPayload(array $payload): self
     {
-        Assert::keyExists($payload, 'color');
-        Assert::keyExists($payload, 'name');
-        Assert::keyExists($payload, 'url');
-        Assert::stringNotEmpty($payload['color']);
-        Assert::regex($payload['color'], '/^[0-9a-f]{6}$/i');
```

Comment:
you've lost the regex check

Reply:
ops, fixed.

and added a test to ensure it won't be lost again.

Reply:
diff: https://github.com/laminas/automatic-releases/compare/89ecec8ce34b43e1e6bdd0f876e5f5a4dbaa4a64..6ad991b65aa5b9d52febf91dc9ca3f812a80952e